Kickstarter-iOS/Views/Controllers/CreatePasswordViewController.swift

```diff
@@ -0,0 +1,115 @@
+import Library
+import Prelude
+import UIKit
+
+private struct ReuseIdentifier {
+  static let cell = "SettingsInputCell"
+  static let header = "SettingsHeaderView"
+  static let footer = "SettingsFooterView"
+}
+
+private enum CreatePasswordRow: CaseIterable {
+  case newPassword
+  case confirmNewPassword
+}
+
+extension CreatePasswordRow {
+  var title: String {
+    switch self {
+    case .newPassword: return Strings.New_password()
+    case .confirmNewPassword: return Strings.Confirm_password()
+    }
+  }
+
+  var placeholder: String {
+    return Strings.signup_input_fields_password()
+  }
+}
+
+class CreatePasswordViewController: UITableViewController {
+  // MARK: - Accessors
+
+  private lazy var rightBarButtonItem: UIBarButtonItem = {
+    return UIBarButtonItem(barButtonSystemItem: .save, target: nil, action: nil)
+      |> \.isEnabled .~ false
+  }()
+
+  // MARK: - Lifecycle
+
+  override func viewDidLoad() {
+    super.viewDidLoad()
+
+    _ = self
+      |> \.title %~ { _ in Strings.Create_password() }
+
+    _ = self.navigationItem
+      |> \.rightBarButtonItem .~ self.rightBarButtonItem
+
+    _ = self.tableView
+      |> \.allowsSelection .~ false
+      |> \.separatorInset .~ .zero
+
+    if #available(iOS 11, *) { } else {
+      let estimatedSectionFooterHeight: CGFloat = 44
+      let estimatedSectionHeaderHeight: CGFloat = 100
+      let estimatedRowHeight: CGFloat = 44
+      let height = UITableView.automaticDimension
+
+      _ = self.tableView
+        |> \.estimatedSectionFooterHeight .~ estimatedSectionFooterHeight
+        |> \.estimatedSectionHeaderHeight .~ estimatedSectionHeaderHeight
+        |> \.estimatedRowHeight .~ estimatedRowHeight
+        |> \.sectionFooterHeight .~ height
+        |> \.sectionHeaderHeight .~ height
+        |> \.rowHeight .~ height
+    }
+
+    self.tableView.register(
+      SettingsGroupedHeaderView.self, forHeaderFooterViewReuseIdentifier: ReuseIdentifier.header
```

Comment:
Since we are moving away from storyboards, I think we could create a UITableView extension with a function similar to the one we have on `Nib.swift` to register headers and footers. Something like:
```
public func registerHeaderFooter(_ headerFooter: UITableViewHeaderFooterView.Type,
                                   inBundle bundle: Bundle = .framework) {
    self.register(
      headerFooter,
      forHeaderFooterViewReuseIdentifier: headerFooter
        .description()
        .components(separatedBy: ".")
        .dropFirst()
        .joined(separator: "."))
  }
```

So we could use `self.tableView.registerHeaderFooter(SettingsGroupedHeaderView.self)` instead. For this, would just need to make sure that the identifier is the class name `"SettingsGroupedHeaderView"`,`"SettingsGroupedFooterView"`, etc. For this, we could go even a little further and create a protocol `Identifiable` (or whatever) with a `identifier` property that gives us the identifier automatically. 

But we can discuss this later on.